tests_new/integration_tests/core/vpc/conftest.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from integration_tests.core.vpc.queries import create_network, delete_network
+
+
+@pytest.fixture(scope='function')
+def network1(client1, group1, session_env1, session_id):
+    network = create_network(
```

Comment:
minor: should we wrap this in `try/finally` block similar to some other fixtures to handle the case of if network is not created successfully?